.github/workflows/main.yml

```diff
@@ -0,0 +1,96 @@
+name: Main
+
+on:
+  push:
+    branches: [ main, wip ]
+  pull_request:
+    branches: [ main ]
+  schedule:
+    # Every twelve (12) hours.
+    # @see https://crontab.guru/every-12-hours
+    - cron: 0 */12 * * *
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: "${{ matrix.os }}-latest"
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu ]
+        php: [ 8.1, 8.2, 8.3 ]
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4 # https://github.com/marketplace/actions/checkout
+
+      - name: Set up PHP CLI
+        uses: shivammathur/setup-php@v2 # https://github.com/marketplace/actions/setup-php-action
+        with:
+          php-version: "${{ matrix.php }}"
+
+      - name: Create project
+        run: |
+          set -o xtrace
+
+          # Prepare the target directory.
+          sudo mkdir -p /var/www
+          sudo rm -rf /var/www/*
+          sudo chmod 777 /var/www
```

Comment:
I'm surprised this works; usually `composer create-project` exits with an error if the target directory already exists.